src/frontends/tensorflow/tests/convert_tricky_models.cpp

```diff
@@ -501,3 +501,80 @@ TEST_F(TransformationTestsF, SplitInFunction) {
+                                                  score_threshold,
+                                                  soft_nms_sigma,
+                                                  NonMaxSuppression::BoxEncodingType::CORNER,
+                                                  false,
+                                                  i32);
+
+        // compute the first output - selected_indices
+        auto slice_const_one = make_shared<Constant>(i32, Shape{1}, 1);
+        auto slice_const_one_2 = make_shared<Constant>(i32, Shape{1}, 1);
+        auto slice_const_two = make_shared<Constant>(i32, Shape{1}, 2);
+        auto slice_const_three = make_shared<Constant>(i32, Shape{1}, 3);
+        auto slice =
+            make_shared<Slice>(nms->output(0), slice_const_two, slice_const_three, slice_const_one, slice_const_one_2);
+        Output<Node> selected_indices = make_shared<Squeeze>(slice, slice_const_one_2);
+        auto const_ten = make_shared<Constant>(i32, Shape{}, 10);
+        selected_indices = make_shared<Add>(selected_indices, const_ten);
+        auto const_minus_one = make_shared<Constant>(i32, Shape{1}, -1);
+        selected_indices = make_shared<Reshape>(selected_indices, const_minus_one, false);
+
+        // compute the second output - selected_scores
+        auto slice2_const_one = make_shared<Constant>(i32, Shape{1}, 1);
+        auto slice2_const_one_2 = make_shared<Constant>(i32, Shape{1}, 1);
+        auto slice2_const_two = make_shared<Constant>(i32, Shape{1}, 2);
+        auto slice2_const_three = make_shared<Constant>(i32, Shape{1}, 3);
+        auto slice2 =
+            make_shared<Slice>(nms->output(1), slice_const_two, slice_const_three, slice_const_one, slice_const_one_2);
+        Output<Node> selected_scores = make_shared<Squeeze>(slice2, slice_const_one_2);
+        selected_scores = make_shared<ConvertLike>(selected_scores, mul);
+        auto const_five = make_shared<Constant>(f32, Shape{}, 5);
+        selected_scores = make_shared<Multiply>(selected_scores, const_five);
+        selected_scores = make_shared<Convert>(selected_scores, i32);
+        auto const_minus_one_2 = make_shared<Constant>(i32, Shape{1}, -1);
+        selected_scores = make_shared<Reshape>(selected_scores, const_minus_one_2, false);
+
+        // compute the third output - valid_outputs
+        Output<Node> valid_outputs = make_shared<Squeeze>(nms->output(2));
+        auto const_five_int32 = make_shared<Constant>(i32, Shape{}, 5);
+        valid_outputs = make_shared<Add>(valid_outputs, const_five_int32);
+        auto const_minus_one_3 = make_shared<Constant>(i32, Shape{1}, -1);
+        valid_outputs = make_shared<Reshape>(valid_outputs, const_minus_one_3, false);
+
+        // concatenate all outputs
+        auto concat = make_shared<Concat>(OutputVector{selected_indices, selected_scores, valid_outputs}, 0);
+
+        model_ref = make_shared<Model>(OutputVector{concat}, ParameterVector{boxes, scores});
```

Comment:
Looks quite bulky. Do we really need such a big test to verify output names for NMS? Can we use something more compact?

Reply:
I will simplify this. Thanks.

Reply:
@slyalin, simplified as much as I can. Remained nodes created by NMSV5 translator to get aligned output formats with TF.